src/components/MintDialog/pages/NativeMint/NativeMint.tsx

```diff
@@ -116,6 +135,30 @@ export const NativeMint: FC<NativeMintProps> = ({
             Buy with credit card
           </Button>
         ) : null}
+        {dropDataSuffix && (
```

Comment:
I think there was an ask to move this above the buttons, let me check

Reply:
I designed it using the [Figma](https://www.figma.com/file/dSJVPE1taRWLqdqIJCPZAA/Onchain-Summer?type=design&node-id=4352-27373&mode=dev) file